.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: scala
+scala:
+  - 2.12.3
+jdk:
+  - oraclejdk8
+sudo: required
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/launchers
+before_cache:
+  # Ensure changes to the cache aren't persisted
+  - rm -rf $HOME/.ivy2/cache/com/lightbend/rp/sbtreactiveapp
+  # Delete all ivydata files since ivy touches them on each build
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" | xargs rm
+script:
+  - sbt test
+  - sbt scripted
```

Comment:
`sbt ^test ^scripted` will test both SBT 1.0 and 0.13 versions of the plugin.

Reply:
Ah, yes - thanks for catching that.